airflow/cli/commands/connection_command.py

```diff
@@ -67,6 +69,52 @@ def connections_list(args):
         print(msg)
 
 
+def _format_connections(conns: List[Connection], fmt: str):
+    connections_json = {}
+    for conn in conns:
+        connections_json[conn.conn_id] = {
+            'conn_type': conn.conn_type,
+            'host': conn.host,
+            'port': conn.port,
+            'is_encrypted': conn.is_encrypted,
+            'is_extra_encrypted': conn.is_encrypted,
+            'extra': json.loads(conn.extra) if conn.extra else None,
+        }
+
+    if fmt == 'yaml':
+        return yaml.dump(connections_json)
+
+    if not fmt or fmt == 'json':
+        return json.dumps(connections_json)
+
+
+def connections_export(args):
+    """Exports all connections to a file"""
+    allowed_formats = ['yaml', 'json']
```

Comment:
Should we also support `.env` file?  I think this might be helpful as this command might be the easiest way to create a connection as a URL,